### ISSUE TITLE
Fix Request: pmd7 HttpClientBuilderWithoutTimeouts false positive

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#402]
```
class Issue402 {
   private RequestConfig reqConfig;

    @Override
    public boolean foo() {
        return reqConfig.bar(); // <--- false positive
    }
}
```
